src/Stream/StreamFactoryInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Psr\Http\Factory\Stream;
+
+use Psr\Http\Message\StreamInterface;
+
+interface StreamFactoryInterface
```

Comment:
I still think this interface is duplicating `StringStreamFactoryInterface` - both create a string-based stream, so the implementations are going to be identical, only the initialization differs, and you can cover that by adding a default value of `''` to the argument.